css/main.css

```diff
@@ -0,0 +1,62 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.container {
+  margin: 0 24px;
+}
+
+header p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.section-1 {
+  background-image: url(../images/background.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  min-height: 80vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
```

Comment:
@vechicin not sure till now about it I am just trying to design my site for mobile users as I follow a bottom-up approach and still I am in the early stages, anyway I changed some configurations inside my CSS file I made my background-size to 100% instead of value contain, also added top margin for header if you still see errors here don't hesitate to comment here, please